app/controllers/users.js

```diff
@@ -44,6 +44,99 @@ exports.findUser = (req, res) => {
   });
 };
 
+exports.getFriends = (req, res) => {
+  const userId = req.body.userId;
+  try {
```

Comment:
Thanks for pointing this out, i have replaced all the try, catch blocks with the appropraite response messages